hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/MonitorServiceConfigurationBuilder.java

```diff
@@ -1377,6 +1382,33 @@ private static String getObjectName(ModelNode modelNode, OperationContext contex
         }
     }
 
+    private static List<OperationParam> getOpParamListFromOpNode(ModelNode modelNode) throws OperationFailedException {
+
+        List<OperationParam> ret = new ArrayList<>();
+
+        ModelNode params = modelNode.get("param");
+        if (params.getType()==ModelType.UNDEFINED) {
+            return Collections.emptyList();
+        }
+        List<Property> ps = params.asPropertyList();
+        for (Property prop : ps) {
+            String name = prop.getName();
+            ModelNode m = prop.getValue();
+            String typ = m.get("type").asString();
+            String desc = m.get("description").asString();
+
+            OperationParam operationParam = new OperationParam(name,typ,desc);
```

Comment:
trivial : java format dictates spaces after "," in param lists. I can reformat using the eclipse java formatter before finally merging to clean these all up. I'm sure I missed some.